fb-negotiation/sessions/rulebased_session.py

```diff
@@ -0,0 +1,380 @@
+import random
+from cocoa.core.entity import is_entity
+from session import Session
+from core.tokenizer import tokenize
+import copy
+import sys
+import re
+
+class RulebasedSession(object):
+    @staticmethod
+    def get_session(agent, kb, lexicon=None):
+        if kb.facts['Role'] == 'first':
+            return FirstRulebasedSession(agent, kb, lexicon)
+        elif kb.facts['Role'] == 'second':
+            return SecondRulebasedSession(agent, kb, lexicon)
+        else:
```

Comment:
Why do you need two Rulebased sessions? The two agents are symmetric so we only need one bot right?